### ISSUE TITLE
player.disconnect() issue

### ISSUE THREAD

**Issue by Spimy** [PythonistaGuild/Wavelink#37]
```Traceback (most recent call last):
  File "C:\Users\acer\AppData\Local\Programs\Python\Python37-32\lib\site-packages\wavelink\websocket.py", line 144, in process_data
    await self._node.on_event(event)
  File "C:\Users\acer\AppData\Local\Programs\Python\Python37-32\lib\site-packages\wavelink\node.py", line 152, in on_event
    await event.player.hook(event)
AttributeError: 'NoneType' object has no attribute 'player'
```
I get this when I do `await player.disconnect()` to disconnect the bot/player from the voice channel and I don't know why this happens.

**Comment by EvieePy** [comment]
Sorry for the late reply.

If you are still having an issue could you please provide a minimum repro example?
Thanks

**Comment by EvieePy** [comment]
Fixed thanks 👍 36de6a8
